workflows.py

```
@@ -10,22 +10,13 @@
 from hivemind_etl.mediawiki.workflows import (
     MediaWikiETLWorkflow,
 )
+from hivemind_etl.simple_ingestion.pipeline import (
+    IngestionWorkflow,
```

Ruff (0.8.2)

`hivemind_etl.simple_ingestion.pipeline.IngestionWorkflow` imported but unused

Remove unused import: `hivemind_etl.simple_ingestion.pipeline.IngestionWorkflow`

(F401)